main.js

```diff
@@ -57,18 +78,7 @@ function displayNumbers(randomNumbers, currentScore) {
     const numbersOnScreen = randomNumbers.length;
```

Comment:
can we change the name of this variable. its not the `numbersOnScreen` its the amount of numbers to generate